disttask/framework/dispatcher/dispatcher.go

```diff
@@ -40,6 +40,8 @@ const (
 	MaxSubtaskConcurrency = 256
 	// DefaultLiveNodesCheckInterval is the tick interval of fetching all server infos from etcd.
 	DefaultLiveNodesCheckInterval = 2
+	// defaultHistorySubtaskTableGcInterval is the interval of gc history subtask table.
+	defaultHistorySubtaskTableGcInterval = 10 * time.Minute
```

Comment:
If this is for history table gc, I think 1 time per day is ok.

Reply:
LGTM